js/core.js

```diff
@@ -308,13 +317,23 @@
          * @return {Promise} a promise that returns json object or snaptime if resolved or
          *      {@link ERMrest.ERMrestError} if rejected
          */
-        currentSnaptime: function (action) {
+        currentSnaptime: function (header) {
```

Comment:
Just for consistency with the rest of functions in ermrestjs let's rename the parameter to `contextHeaderParams `. Please update the jsdoc as well.